packages/shared/lib/core/network/interfaces/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './client-options.interface'
```

Comment:
These should be alphabetized.

Reply:
There was a few issues with some types / enums used with each other not exporting in the correct order